agent/stats/queue.go

```diff
@@ -305,19 +272,6 @@ func getInt64WithOverflow(uintStat uint64) (int64, int64) {
 type getUsageFloatFunc func(*UsageStats) float64
 type getUsageIntFunc func(*UsageStats) uint64
 
-func (queue *Queue) resetThresholdElapsed(timeout time.Duration) bool {
-	queue.lock.RLock()
-	defer queue.lock.RUnlock()
-	duration := time.Since(queue.lastResetTime)
-	return duration.Seconds() > timeout.Seconds()
-}
-
-func (queue *Queue) enoughDatapointsInBuffer() bool {
-	queue.lock.RLock()
-	defer queue.lock.RUnlock()
-	return len(queue.buffer) >= minimumQueueDatapoints
-}
-
 // getCWStatsSet gets the stats set for either CPU or Memory based on the
 // function pointer.
 func (queue *Queue) getCWStatsSet(f getUsageFloatFunc) (*ecstcs.CWStatsSet, error) {
```

Comment:
f is not an informative name, getUsageFloat maybe?

Reply:
I didn't change this, did you want me to rename it?

Reply:
if you are making changes for below comment, might as well? In the code in this file, f can be either getUsageIntFunc or getUsageFloatFunc and that's confusing.

Reply:
ya sure i'll change it

Reply:
done